src/routes/github/github-oauth-router.ts

```diff
@@ -127,7 +127,9 @@ const GithubOAuthCallbackGet = async (req: Request, res: Response, next: NextFun
 export const GithubAuthMiddleware = async (req: Request, res: Response, next: NextFunction) => {
 	try {
 		const { githubToken, gitHubUuid } = req.session;
-		const { jiraHost, gitHubAppId, gitHubAppConfig } = res.locals;
+		const { jiraHost, gitHubAppConfig } = res.locals;
+		const gitHubAppId = res.locals.gitHubAppId || gitHubAppConfig.id;
```

Comment:
gitHubAppId isn't available from res.locals in all paths. For the subscription portion of the flow gitHubAppConfig, but not gitHubAppId

Reply:
In the `github-server-app-middlware` doesn't generate `gitHubAppConfig.id`, it generates `gitHubAppConfig.gitHubAppId`. That's the same one as `res.locals.gitHubAppId`.

Reply:
Can use whichever one.